draft-ietf-webtrans-http3.md

```diff
@@ -709,3 +714,26 @@ Notes:
 {: spacing="compact"}
 
 --- back
+
+# Changelog
+
+## Changes between draft versions 02 and 06
+
+The following changes make the draft-02 and draft-06 versions of this protocol
+incompatible:
+
+* draft-06 requires SETTINGS_WEBTRANSPORT_MAX_SESSIONS instead of
```

Comment:
`SETTINGS_ENABLE_WEBTRANSPORT` is not gone yet, but it would be nice if it were!